run-build-functions.sh

```diff
@@ -898,7 +890,7 @@ set_go_import_path() {
     local importPath="$GOPATH/src/$GO_IMPORT_PATH"
     local dirPath="$(dirname $importPath)"
 
-    rm -rf $dirPath
+    rm -rf $importPath
```

Comment:
Looking at this again, I'm actually not sure this is removing the correct directory anymore. This was removing `$PWD/dirname`, but it's now removing `$GOPATH/src/$GO_IMPORT_PATH` instead. I think this should remain as `rm -rf $dirpath` like it was before.

Reply:
Nevermind - I changed that on purpose and I'm now remembering why. Before, it was removing siblings of the `$importPath` and now it's *only* removing the `$importPath` itself.